stable/mongodb/values.yaml

```diff
@@ -116,6 +116,9 @@ clusterDomain: cluster.local
 
 ## Kubernetes service type
 service:
+  ## Specify an explicit service name. This is not recommended in general and therefore
```

Comment:
Remove this part of the comment as it is already explained in the `_helpers.tpl` 
```diff 
- ## Specify an explicit service name. This is not recommended in general and therefore
- ## disabled by default. The service name will be set to the fullname of the chart.
+ ## Specify an explicit service name.
```